tests/test_interface.rb

```diff
@@ -980,19 +980,15 @@ def test_interface_fabric_forwarding_anycast_gateway
     int.fabric_forwarding_anycast_gateway = false
     refute(int.fabric_forwarding_anycast_gateway)
 
-    # 4. Removing fabric forwarding anycast gateway mac
-    foo.anycast_gateway_mac = foo.default_anycast_gateway_mac
-    assert_raises(RuntimeError) { int.fabric_forwarding_anycast_gateway = true }
-
-    # 5. Removing feature fabric forwarding
-    config('no feature fabric forwarding')
-    assert_raises(RuntimeError) { int.fabric_forwarding_anycast_gateway = true }
-
-    # 6. Attempting to configure on a non-vlan interface
+    # 4. Attempt to configure on a non-vlan interface
     nonvlanint = create_interface
     assert_raises(RuntimeError) do
       nonvlanint.fabric_forwarding_anycast_gateway = true
     end
+
+    # 5. Removing fabric forwarding anycast gateway mac
```

Comment:
Nit: `#5. Attempt to set 'fabric forwarding anycast gateway` while the overlay gateway mac is not set.`